Buttler.Backend/Buttler.Data/ApplicationDbContext.cs

```diff
@@ -6,12 +6,14 @@ namespace Buttler.Data;
 public class ApplicationDbContext: DbContext
 {
     public DbSet<Report> Reports { get; set; }
+    public DbSet<User> Users { get; set; }
 
     protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         => optionsBuilder.UseNpgsql("Host=buttler1.c0zpqoi4e1d8.eu-north-1.rds.amazonaws.com;Database=buttler_db;Username=postgres;Password=buttlerpass");
     
     protected override void OnModelCreating(ModelBuilder modelBuilder)
     {
         modelBuilder.Entity<Report>().HasKey(report => report.ReportId);
```

Comment:
Remember to add the username as a foreign key now that you have a user as being a part the report. This can be done using the HasOne method since a report has one user.